Lib/unittest/test/testmock/testmock.py

```diff
@@ -558,6 +558,16 @@ def test_wraps_calls(self):
         real.assert_called_with(1, 2, fish=3)
 
 
+    def test_wraps_prevents_automatic_creation_of_mocks(self):
+        class Real(object):
+            attribute = Mock()
```

Comment:
Why is this a mock? Is it important that `Real` has an `attribute` or could this just be:

```
class Real(object): pass
```

?

Reply:
That was indeed a bad *copy-paste*. 😞 .